corehq/apps/cleanup/management/commands/undelete_docs.py

```diff
@@ -13,7 +14,12 @@ def get_deleted_doc(db, doc_id, rev):
     start = res['_revisions']['start']
     ids = res['_revisions']['ids']
     prev_revision = "{}-{}".format(start-1, ids[1])
-    doc = db.get(doc_id, rev=prev_revision)
+    try:
+        doc = db.get(doc_id, rev=prev_revision)
+    except ResourceNotFound:
+        res.pop('_deleted')
+        res.pop('_revisions')
+        doc = res
```

Comment:
This looks like cloudant-specific behavior - locally, I don't get the full doc in the response.